docs/features/wired-mesh.rst

```diff
@@ -85,3 +85,33 @@ Disable Mesh-on-LAN::
 Please note that this configuration has changed in Gluon 2016.1. Using
 the old commands on 2016.1 and later will break the corresponding options
 in the *Advanced settings*.
+
+Mesh on other interfaces
+========================
+In some cases, you may want to mesh on more interfaces than only LAN or WAN.
+
+This is especially useful if you have multiple other nodes connected to a central node,
+and don't want them to mesh with each other, but only the central node.
+
+In this case, you may need to isolate them via VLANs on the device switch.
```

Comment:
This sentence sounds a bit weird on its own, it should be merged into the paragraph above.